lego/apps/emojis/models.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.postgres import fields
+from django.db import IntegrityError, models
+from rest_framework.exceptions import ValidationError
+
+from lego.apps.emojis.permissions import EmojiPermissionHandler
+from lego.utils.managers import BasisModelManager
+from lego.utils.models import BasisModel
+
+
+class Emoji(BasisModel):
+    id = models.CharField(max_length=40, primary_key=True)
```

Comment:
Maybe this should be named something more descriptive? I wouldn't know from the name `id` that this field contains e.g. `":smile:"`

Reply:
I agree somewhat, in that case we just need a descriptive name, perhaps `short_code` as it was previous named. I would however like some help with handling the `byId` in the frontend, as I'm not sure how to handle index it with other fields than `id`.

Reply:
Like this (using tag as the pk in normalizer)?
https://github.com/webkom/lego-webapp/blob/master/app/reducers/index.js#L219